samples/aws/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
```

Comment:
@karolz-ms we need to be able to replicate this with dcp. Are all of the primitives there?

cc @mitchdenny